pkgs/tools/security/gnu-pw-mgr/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, lib, fetchurl, autogen, gnulib }:
+
+stdenv.mkDerivation {
+  name = "gnu-pw-mgr-2.3.2";
+  src = fetchurl {
+    url = "http://ftp.gnu.org/gnu/gnu-pw-mgr/gnu-pw-mgr-2.3.2.tar.xz";
+    sha256 = "0x60g0syqpd107l8w4bl213imy2lspm4kz1j18yr1sh10rdxlgxd";
+  };
+
+  buildInputs = [
+    autogen
+    gnulib
+  ];
+
+  meta = with lib; {
+    homepage = "https://www.gnu.org/software/gnu-pw-mgr/";
+    description = "GNU gnu-pw-mgr is a password manager designed to make it easy to reconstruct difficult passwords when they are needed, while limiting the risk of attack.";
+    license = with licenses; [ gpl3Plus lgpl3Plus ];
+    platforms = platforms.all;
+    maintainers = [ ];
```

Comment:
Do you want to maintain this yourself? In this case please add a commit before the current one adding yourself to https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix.

Reply:
That makes sense, done. Thanks for taking the time to review this PR!